Themes/default/Reminder.template.php

```diff
@@ -146,42 +123,29 @@ function template_set_password()
-		<div class="tborder login">
-			<div class="cat_bar">
-				<h3 class="catbg">', Lang::$txt['authentication_reminder'], '</h3>
+	<form action="', Config::$scripturl, '?action=reminder;sa=secret2" method="post" accept-charset="', Utils::$context['character_set'], '" name="creator" class="login">
+		<div class="cat_bar">
+			<h3 class="catbg">', Lang::$txt['authentication_reminder'], '</h3>
+		</div>
+		<div class="windowbg form_grid">
+			<p class="descbox">', Lang::$txt['enter_new_password'], '</p>
+			<label>', Lang::$txt['secret_question'], ':</label>
+			<div>', Utils::$context['secret_question'], '</div>
+			<label>', Lang::$txt['secret_answer'], ':</label>
+			<div><input type="text" name="secret_answer" autofocus size="22"></div>
+			<label>', Lang::$txt['choose_pass'], ': </label>
+			<div>
+				<input type="password" name="passwrd1" data-autov="pwmain" size="22">
+			</div>
+			<label>', Lang::$txt['verify_pass'], ': </label>
+			<div>
+				<input type="password" name="passwrd2" data-autov="pwverify" size="22">
 			</div>
-			<div class="roundframe">
-				<p class="smalltext">', Lang::$txt['enter_new_password'], '</p>
-				<dl>
-					<dt>', Lang::$txt['secret_question'], '</dt>
-					<dd>', Utils::$context['secret_question'], '</dd>
-					<dt>', Lang::$txt['secret_answer'], '</dt>
-					<dd><input type="text" name="secret_answer" size="22"></dd>
-					<dt>', Lang::$txt['choose_pass'], '</dt>
-					<dd>
-						<input type="password" name="passwrd1" id="smf_autov_pwmain" size="22">
-						<span id="smf_autov_pwmain_div" style="display: none;">
-							<span id="smf_autov_pwmain_img" class="main_icons invalid"></span>
-						</span>
-					</dd>
-					<dt>', Lang::$txt['verify_pass'], '</dt>
-					<dd>
-						<input type="password" name="passwrd2" id="smf_autov_pwverify" size="22">
-						<span id="smf_autov_pwverify_div" style="display: none;">
-							<span id="smf_autov_pwverify_img" class="main_icons valid"></span>
-						</span>
-					</dd>
-				</dl>
-				<div class="auto_flow">
-					<input type="submit" value="', Lang::$txt['save'], '" class="button">
-					<input type="hidden" name="uid" value="', Utils::$context['remind_user'], '">
-					<input type="hidden" name="', Utils::$context['session_var'], '" value="', Utils::$context['session_id'], '">
-					<input type="hidden" name="', Utils::$context['remind-sai_token_var'], '" value="', Utils::$context['remind-sai_token'], '">
-				</div>
-			</div><!-- .roundframe -->
-		</div><!-- .login -->
+			<input type="submit" value="', Lang::$txt['save'], '" class="button">
+			<input type="hidden" name="uid" value="', Utils::$context['remind_user'], '">
+			<input type="hidden" name="', Utils::$context['session_var'], '" value="', Utils::$context['session_id'], '">
+			<input type="hidden" name="', Utils::$context['remind-sai_token_var'], '" value="', Utils::$context['remind-sai_token'], '">
+		</div>
 	</form>
 	<script>
 		var regTextStrings = {
```

Comment:
If it is not possible to use `const`, then it is better to use `let`, but not `var`. `var` is already too obsolete. Similarly for other use cases. Using var is justified if you need to write code for old browsers of IE 11 or Opera mini.